engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java

```diff
@@ -827,6 +827,16 @@ private void getAllPortsWithAncestor(TypeGraphVertex portVertex, TypeGraphVertex
     }
   }
 
+  private static boolean contains(List<CompactFieldNode> list, CompactFieldNode vertex)
```

Comment:
Can we have another name for this. "contains" seems too common and not very clear here